### ISSUE TITLE
Bleed-through due to screen

### ISSUE THREAD

**Issue by IreneLenzi1994** [MouseLand/suite2p#812]
Hello!

We are using suite2p to process 2 photon data aquired while mice perform a visual detection task.
Since we present visual stimuli on 2 screens, we have quite a bit of problems with 
bleed-through in the signal when the stimulus is presented ipsilaterally to the imaging location.

Do you maybe have some tips about how to tackle this issues and improve the signal extraction? 
Maybe some specific settings for background correction?

Thanks, Irene

**Comment by trose-neuro** [comment]
- light shield
- synchronize LED backlight of screen to scanner turnaround

**Comment by carsen-stringer** [comment]
I agree with @trose-neuro, this should be done on the experimental side not post-hoc. the traces are baselined in the deconvolution step, and the neuropil subtraction step should help to some extent but I would always be wary that the signals are from the stimulus and not the neurons
